src/arbeidssøkerskjema/steg/1-Spørsmål.tsx

```diff
@@ -38,6 +39,14 @@ const Spørsmål: FC = () => {
   const { skjema, settSkjema } = useSkjema();
   const [arbeidssøker, settArbeidssøker] = React.useState(skjema.arbeidssøker);
 
+  useEffect(() => {
+    logEvent('sidevisning', {
```

Comment:
Kunne vi laget wrappere som gjør det litt mindre 'string'-basert? 
`logArbeidssokerSkjema('forside')`
`logBarnetilsyn('...sidenavn...')`
`logSkolepenger('...sidenavn...')`
`logOvergangsstønad('...sidenavn...')`

Så kan de respektive 'log...'-funksjonene sørge for at `'sidevisning',  team: 'familie' og app: '...'` settes riktig?

Reply:
Og kanskje egen hook så slipper vi useEffect(...) overalt? :)